Arrays/2016MaximumDifferenceBetweenIncreasingElements.py

```
@@ -0,0 +1,32 @@
+class Solution:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)